src/scripts/Package-Toolkit.ps1

```diff
@@ -88,10 +88,34 @@ $templates = Get-ChildItem $relDir -Directory `
         }
     }
 
-    # Copy azuredeploy.json to docs/deploy folder
     Write-Verbose "Updating $($path.Name) deployment file in docs..."
-    Copy-Item "$path/azuredeploy.json" "$deployDir/$($path.Name)-$version.json"
-    Copy-Item "$path/azuredeploy.json" "$deployDir/$($path.Name)-latest.json"
+
+    $packageManifestPath = "$path/package-manifest.json"
+    if (Test-Path $packageManifestPath)
```

Comment:
nit: Would it be better to define a fallback manifest instead of just falling back to the previous logic? That way, we always work with a manifest, we just define a default one. I'm okay to not do that or maybe just add a TODO comment. Just a thought that crossed my mind.

Reply:
Yes, I agree. I was about to include this change, but then I thought: where do we define the fallback manifest? Inline or as a file placed somewhere in the src structure? As I didn't want to make this decision now, I am leaving a TODO comment for now.